packages/node-opentelemetry/examples/dummy.js

```diff
@@ -120,13 +145,70 @@ app.use((req, res, next) => {
 
 app.use(express.json());
 
-app.post('/dump', (req, res) => {
-  const body = req.body;
-  console.log(body);
-  res.send('Hello World');
+app.get('/instruments', async (req, res) => {
+  await Promise.all([
+    // FIXME: not working
+    initInstrumentationTest('mongodb', async () => {
+      const mongoClient = new MongoClient('mongodb://localhost:27017');
+      await mongoClient.connect();
+      const db = mongoClient.db('hyperdx');
+      await db.collection('teams').find({}).toArray();
+    }),
+    // FIXME: not working
+    initInstrumentationTest('mongoose', async () => {
+      await mongoose.connect('mongodb://localhost:27017/hyperdx');
+      const User = mongoose.model('User', {
+        name: String,
+      });
+      await User.find({}, { name: 1 }).exec();
+    }),
+    initInstrumentationTest('ioredis', async () => {
+      const redis = new IORedis({
+        host: 'localhost',
+        port: 6379,
+      });
+      await redis.set('foo', 'bar');
+      await redis.get('foo');
+    }),
+    initInstrumentationTest('redis', async () => {
+      const redis = await Redis.createClient({
+        host: 'localhost',
+        port: 6379,
+      })
+        .on('error', (error) => {
+          logger.error('Redis error', error);
+        })
+        .connect();
+      await redis.set('foo1', 'bar1');
+      await redis.get('foo1');
+      await redis.disconnect();
+    }),
+    initInstrumentationTest('dns', async () => {
+      return new Promise((resolve, reject) => {
+        dns.lookup('example.com', (err, address, family) => {
+          if (err) {
+            reject(err);
+          } else {
+            resolve(address);
+          }
+        });
+      });
+    }),
+    initInstrumentationTest('https', async () => {
+      return new Promise((resolve, reject) => {
+        https.get('https://example.com', (res) => {
+          res.on('data', (data) => {
+            resolve(data.toString());
+          });
+        });
+      });
+    }),
```

Comment:
error handling?

Reply:
it doens't matter haha. just need to check if https works